### PR TITLE
Bug Fix #1219 : Check style failure in windows machine

### LINKED ISSUE #1219: Checkstyle failure in windows machine
### Description:
There is Checkstyle failure when building the project.

### Steps to reproduce:

1. Use windows machine.
2. git clone the MGW repository.
3. Run mvn clean install.
3. window build fails with checkstyle issues.

### Affected Product Version:
MGW-3.1.1

### Environment details (with versions):
- OS: window
- Client:
- Env (Docker/K8s):

---

### PR DESCRIPTION
**Pull request by tharmini**
### Purpose
Resolved check style failure in windows machine.

### Issues
#1219 
#984

### Tested environments
Tested in Windows

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIDetailedDTO.java [hunk 1]
```diff
@@ -107,7 +107,7 @@
     private String provider = null;
 
     /**
-     * Swagger definition of the APIDetailedDTO which contains details about URI templates and scopes\n
+     * Swagger definition of the APIDetailedDTO which contains details about URI templates and scopes\n.
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIDetailedDTO.java:110]:
```suggestion
     * Swagger definition of the APIDetailedDTO which contains details about URI templates and scopes.
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIDetailedDTO.java [hunk 2]
```diff
@@ -121,7 +121,7 @@ public void setApiDefinition(String apiDefinition) {
 
 
     /**
-     * WSDL URL if the APIDetailedDTO is based on a WSDL endpoint\n
+     * WSDL URL if the APIDetailedDTO is based on a WSDL endpoint\n.
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIDetailedDTO.java:124]:
```suggestion
     * WSDL URL if the APIDetailedDTO is based on a WSDL endpoint.
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIDetailedDTO.java [hunk 3]
```diff
@@ -188,7 +188,7 @@ public void setType(TypeEnum type) {
 
 
     /**
-     * Supported transports for the APIDetailedDTO (http and/or https).\n
+     * Supported transports for the APIDetailedDTO (http and/or https).\n.
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIDetailedDTO.java:191]:
```suggestion
     * Supported transports for the APIDetailedDTO (http and/or https).
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIDetailedDTO.java [hunk 4]
```diff
@@ -331,7 +331,7 @@ public void setEndpointSecurity(APIEndpointSecurityDTO endpointSecurity) {
     }
 
     /**
-     * Comma separated list of gateway environments.\n
+     * Comma separated list of gateway environments.\n.
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIDetailedDTO.java:334]:
```suggestion
     * Comma separated list of gateway environments.
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIDetailedDTO.java [hunk 5]
```diff
@@ -343,7 +343,7 @@ public void setGatewayEnvironments(String gatewayEnvironments) {
     }
 
     /**
-     * Labels of micro-gateway environments attached to the APIDetailedDTO.\n
+     * Labels of micro-gateway environments attached to the APIDetailedDTO.\n.
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIDetailedDTO.java:346]:
```suggestion
     * Labels of micro-gateway environments attached to the APIDetailedDTO.
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIInfoDTO.java [hunk 6]
```diff
@@ -36,7 +36,7 @@
 
 
     /**
-     * UUID of the api registry artifact\n
+     * UUID of the api registry artifact\n.
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIInfoDTO.java:39]:
```suggestion
     * UUID of the api registry artifact.
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIInfoDTO.java [hunk 7]
```diff
@@ -99,7 +99,7 @@ public void setVersion(String version) {
     }
 
     /**
-     * If the provider value is not given, the user invoking the APIDetailedDTO will be used as the provider.\n
+     * If the provider value is not given, the user invoking the APIDetailedDTO will be used as the provider.\n.
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIInfoDTO.java:102]:
```suggestion
     * If the provider value is not given, the user invoking the APIDetailedDTO will be used as the provider.
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIListDTO.java [hunk 8]
```diff
@@ -36,7 +36,7 @@
 
   
   /**
-   * Number of APIs returned.\n
+   * Number of APIs returned.\n.
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIListDTO.java:39]: @VirajSalaka Are these models generated from using some tool?

Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIListDTO.java:39]:
```suggestion
   * Number of APIs returned.
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIListDTO.java [hunk 9]
```diff
@@ -47,7 +47,7 @@ public void setCount(Integer count) {
   }
   
   /**
-   * Link to the next subset of resources qualified.\nEmpty if no more resources are to be returned.\n
+   * Link to the next subset of resources qualified.\nEmpty if no more resources are to be returned.\n.
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIListDTO.java:50]:
```suggestion
   * Link to the next subset of resources qualified.\nEmpty if no more resources are to be returned.
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIListDTO.java [hunk 10]
```diff
@@ -58,7 +58,7 @@ public void setNext(String next) {
   }
   
   /**
-   * Link to the previous subset of resources qualified.\nEmpty if current subset is the first subset returned.\n
+   * Link to the previous subset of resources qualified.\nEmpty if current subset is the first subset returned.\n.
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIListDTO.java:61]:
```suggestion
   * Link to the previous subset of resources qualified.\nEmpty if current subset is the first subset returned.
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/template/service/BallerinaPath.java [hunk 11]
```diff
@@ -146,7 +146,7 @@ public BallerinaPath getDefaultValue() {
 
 
     /**
-     * Method to set the Path level server opeartions to operation level servers,
+     * Method to set the Path level server opeartions to operation level servers.
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/template/service/BallerinaPath.java:149]:
```suggestion
     * Method to set the Path level server opeartions to operation level servers,
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/template/service/BallerinaPath.java [hunk 12]
```diff
@@ -146,7 +146,7 @@ public BallerinaPath getDefaultValue() {
...
      * if operation level servers are not present
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/template/service/BallerinaPath.java:150]:
```suggestion
     * if operation level servers are not present.
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/policy/SubscriptionThrottlePolicyListDTO.java [hunk 13]
```diff
@@ -29,7 +29,7 @@
     private List<SubscriptionThrottlePolicyDTO> list = new ArrayList<SubscriptionThrottlePolicyDTO>();
 
     /**
-     * Number of Subscription Throttling Policies returned.\n
+     * Number of Subscription Throttling Policies returned.\n.
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/policy/SubscriptionThrottlePolicyListDTO.java:32]:
```suggestion
     * Number of Subscription Throttling Policies returned.
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/policy/SubscriptionThrottlePolicyDTO.java [hunk 14]
```diff
@@ -84,8 +84,9 @@ public void setCustomAttributes(List<CustomAttributeDTO> customAttributes) {
 
 
     /**
-     * This indicates the action to be taken when a user goes beyond the allocated quota. If checked, the user's
-     * requests will be dropped. If unchecked, the requests will be allowed to pass through.\n
+     * This indicates the action to be taken when a user goes beyond the allocated quota.
+     * If checked, the user's requests will be dropped. If unchecked,
+     * the requests will be allowed to pass through.\n
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/policy/SubscriptionThrottlePolicyDTO.java:89]:
```suggestion
     * the requests will be allowed to pass through.
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/policy/SubscriptionThrottlePolicyDTO.java [hunk 15]
```diff
@@ -71,7 +71,7 @@ public void setRateLimitTimeUnit(String rateLimitTimeUnit) {
 
 
     /**
-     * Custom attributes added to the Subscription Throttling Policy\n
+     * Custom attributes added to the Subscription Throttling Policy\n.
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/policy/SubscriptionThrottlePolicyDTO.java:74]:
```suggestion
     * Custom attributes added to the Subscription Throttling Policy.
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/policy/ApplicationThrottlePolicyListDTO.java [hunk 16]
```diff
@@ -29,7 +29,7 @@
 
 
     /**
-     * Number of Application Throttling Policies returned.\n
+     * Number of Application Throttling Policies returned.\n.
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/policy/ApplicationThrottlePolicyListDTO.java:32]:
```suggestion
     * Number of Application Throttling Policies returned.
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIDetailedDTO.java (hunk 1 above)
Comment (tharmini) [reply components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIDetailedDTO.java]: fixed

**Pull request merged**